lib/moab/bagger.rb

```diff
@@ -259,7 +259,7 @@ def create_tarfile(tar_pathname = nil)
       rescue
         shell_execute(tar_cmd.sub('--force-local', ''))
       end
-      raise "Unable to create tarfile #{tar_pathname}" unless tar_pathname.exist?
+      raise(MoabRuntimeError, "Unable to create tarfile #{tar_pathname}") unless tar_pathname.exist?
```

Comment:
This one is trivial: why the parens? I don't think we need them, and it'd look a tad bit more idiomatic to my eyes without them (like before). Totally a style preference, so this isn't blocking; I'm just curious why you added parens to each of these.

Reply:
I believed it was easier to read with parens when followed by `unless` or `if`.  Perhaps I overdid it.